package.json

```diff
@@ -85,6 +86,7 @@
     "fs-extra": "9.0.0",
     "husky": "4.2.5",
     "jest": "26.0.1",
+    "jest-axe": "3.4.0",
```

Comment:
I think it should be added as a dependency on the `reakit-test-utils` package too. Otherwise, installing `reakit-test-utils` will not install `jest-axe`.